scripts/qwik-city.ts

```diff
@@ -67,10 +67,10 @@ export async function buildQwikCity(config: BuildConfig) {
         import: './adapters/cloud-run/vite/index.mjs',
         require: './adapters/cloud-run/vite/index.cjs',
       },
-      './adapters/express/vite': {
-        types: './adapters/express/vite/index.d.ts',
-        import: './adapters/express/vite/index.mjs',
-        require: './adapters/express/vite/index.cjs',
+      './adapters/node-server/vite': {
```

Comment:
we need to have two modules pointing to the same new file, to prevent breaking changes